README.md

```diff
@@ -16,102 +18,116 @@ pip install ./ansible-navigator
 ansible-navigator --help
 ```
 
-By default, ansible-navigator uses execution environments, to use ansible-navigator without an execution enviroment,
-ansible is required
+By default, `ansible-navigator` uses execution environments. Execution environments requires container runtimes like [podman](https://github.com/containers/podman). To install container runtimes, please refer respective installation guides.
+
+
+To use `ansible-navigator` without an execution enviroment, Ansible is required.
 
 ```
 pip install ansible
 ansible-navigator --execution-environment false
```

Comment:
This command is not required as it doesn't set EE to false for a subsequent run. To disable EE you point to the configuration doc and set `enabled: Fasle` and can point to https://github.com/ansible/ansible-navigator/blob/main/docs/settings.rst

Reply:
ok.